e2e-tests/native/test/Selector.ts

```diff
@@ -0,0 +1,21 @@
+
+export class Selector {
+    private queries: Array<string> = new Array<string>()
+
+    public ByText(text: string): Selector {
```

Comment:
Not sure but shouldn't this be lowercase?
```suggestion
    public byText(text: string): Selector {
```

Reply:
I suppose the convention is to use uppercase names for selectors.